modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -74,13 +73,13 @@
 	alt_titles = list("Detective", "Forensic Technician", "Private Investigator", "Forensic Scientist")
 
 /datum/job/doctor
-	alt_titles = list("Medical Doctor", "Surgeon", "Nurse")
+	alt_titles = list("Medical Doctor", "Surgeon", "Nurse", "General Practitioner", "Medical Resident", "Physician") //in memorium to "Podiatrist" , killed in it's cradle
 
-/datum/job/engineering_guard
+/datum/job/engineering_guard //see science guard
 
 
 /datum/job/expeditionary_trooper
-
+	alt_titles = list("Expeditionary Trooper", "Vanguard Operative", "Vanguard Pointman", "Expeditionary Field Medic", "Vanguard Marksman", "Expeditionary Combat Technician") //i know vanguard is dead, i'm adding this because i want, in some small way, for my friend, cheshify's hard work and effort to be remembered, even in a tiny way
```

Comment:
Those comments are just kind of pointless, I know the intention behind it and it's commendable, but this kind of comment would fit better in the PR body itself than in the code forever.
```suggestion
	alt_titles = list("Expeditionary Trooper", "Vanguard Operative", "Vanguard Pointman", "Expeditionary Field Medic", "Vanguard Marksman", "Expeditionary Combat Technician")
```